public/app/percona/backup/components/StorageLocations/RemoveStorageLocationModal/RemoveStorageLocationModal.tsx

```diff
@@ -21,6 +25,18 @@ export const RemoveStorageLocationModal: FC<RemoveStorageLocationModalProps> = (
       message={Messages.getDeleteMessage(location?.name || '')}
       onDelete={handleDelete}
     >
+      <Form
+        onSubmit={() => {}}
+        render={() => (
+          <form>
+            <CheckboxField
+              name="force"
+              label="Force"
+              inputProps={{ onInput: e => setForce((e.target as HTMLInputElement).checked) }}
```

Comment:
Maybe it's better to extend the DeleteModal to support this "Force" checkbox since it's a common thing to other places too and then use the Button inside the modal to trigger the onSubmit of the form and that already receives the force value so no need to override onInput event. (just an improvement suggestion, doesn't need to be part of this PR)

Reply:
Right, totally agree with that!
In that case, I'm gonna open a different PR for that, then replacing it.